Parser/Parser.cs

```diff
@@ -67,6 +67,17 @@ NExpr Expression ()
       return expr;
    }
 
+    // arglist = "(" [expression { "," expression } ] ")"
+   NExpr[] ArgList () {
+      Expect (OPEN, "Expecting function call '(' ");
+      List<NExpr> exprs = new ();
+      if (!Peek (CLOSE)) exprs.Add (Expression ());
+      while (Peek (COMMA)) { if (Match (COMMA)) exprs.Add (Expression ()); }

```

Comment:
Isn't this just:
`while (Match (COMMA)) exprs.Add (Expression ());`

Reply:
Done